packages/magika.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,19 @@
+$ErrorActionPreference = 'Continue'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    $toolName = 'magika'
+    $category = 'File Information'
+    $executablePath = $toolName
+    $arguments = " -h"
+    $iconLocation = Join-Path ${Env:WinDir} "system32\cmd.exe" -Resolve
```

Comment:
why is this needed? can we move this to the `VM-Install-Shortcut` helper instead?

Reply:
This one is slightly tricky...

Whenever we use `-consoleApp`, we expect the `executablePath` to point to an executable, which we can then use as an icon.

For this case it's a little different because there is no executable, it's just a command line command so it leads to no useable icon. I can pass in "the path to cmd.exe" here, but I don't think this makes it any better imo and if anything, it's actually worse because it starts to bloat the `Target Path` in the shortcut.

Inside of `VM-Install-Shortcut` I can't set everything that is a `consoleApp` to use `cmd.exe` as it's icon, because that's not always the case, and I can't think of a way that I can test if `executablepath` actually points to a valid "executable" vs "a command" in a way that wouldn't throw an error (by using an assert).

One option I could think of would be to have a different argument in `VM-Install-Shortcut` which would be for `consoleCommand` to fit this case, but I don't know if that's any cleaner than just having `iconLocation` set to `cmd.exe` like how it is here currently, because of how rare of a case this is.

Reply:
Thanks for the detailed explanation @emtuls. The issue is that you are using `magika` as `$executablePath`. I think it is counteractive to not use a path for a variable call `$executablePath`. I suggest  `$executablePath = $(where.exe magika)`. This way we use the binary that was already an icon.

![image](https://github.com/mandiant/VM-Packages/assets/16052290/54ef0f97-1de1-494c-839b-2ed54e2c0332)